CHANGES.txt

```diff
@@ -45,6 +45,9 @@ Breaking Changes
 Changes
 =======
 
+- Statistics for jobs and operations are enabled by default.  If you don't need
+  any statistics, please add ``stats.enabled: false`` to the CrateDB config.
```

Comment:
you can disable them by setting `stats.enabled` to `false`

since there are various ways how to do it (crate.yml, SET statement, cli argument)